reana_client/api/client.py

```diff
@@ -896,7 +939,12 @@ def mv_files(source, target, workflow, access_token):
 
 
 def get_workflow_disk_usage(workflow, parameters, access_token):
-    """Display disk usage workflow."""
+    """Display disk usage workflow.
+
+    :param workflow: name or id which identifies the workflow.
+    :param parameters: additional input parameters and operational options.
```

Comment:
This should be fixed, as these are parameters to customise the return value and they are not workflow parameters. We should probably fix this also in the OpenAPI spec.